src/components/Button.js

```diff
@@ -21,7 +21,7 @@ export default class Button extends React.Component {
 Button.propTypes = {
   name: PropTypes.string.isRequired,
   wide: PropTypes.string.isRequired,
-  color: PropTypes.string,
+  color: PropTypes.bool,
```

Comment:
What you want to make boolean here is the wide property not the color

Reply:
Thanks for pointing it out, it is resolved!